rules/host.bzl

```diff
@@ -23,6 +23,9 @@ def _host_tools_transition_impl(settings, attr):
         "//hw/bitstream/universal:rom": "//hw/bitstream/universal:none",
         "//hw/bitstream/universal:otp": "//hw/bitstream/universal:none",
         "//hw/bitstream/universal:env": "//hw/bitstream/universal:none",
+        # WARNING This is a horrible hack: when we transition to host, we pretend
+        # that this is earlgrey so opentitantool can compile...
```

Comment:
Agree.  Longer term, we probably need to distinguish between chip-specific constants in opentitantool, stick them into something like a hashmap by chip, and add a `--chip=...` cmdline argument to select which set of constants we care about.

Reply:
Yes that's a big issue. I would like to raise it in the SW WG tomorrow, there is clearly some work to be done here. I think the multitop SW PR is a good starting point to generate such a data structure.

Reply:
One route: I could revive my devicetree generator, and opentitantool could consume its output for a description of the system. That would also leave open support for out-of-tree tops (instead of hard-coding entire known chips).